### DISCUSSION TITLE
Load Custom Local Transform Model in config.cfg

### DISCUSSION THREAD

**Discussion by leoitcode** [explosion/spaCy#9767] [Help: Coding & Implementations] [answered]
Hi people! I'm trying to fine-tune with a portuguese BERT trained in legal documents, but this BERT isn't in huggingface hub. At my config.cfg in portuguese setting generated in spaCy web, I got:

`[components.transformer.model]
@architectures = "spacy-transformers.TransformerModel.v1"
name = "neuralmind/bert-base-portuguese-cased"`

This bert-base-portuguese-cased is the default portuguese model on spaCy setting, instead of this I want to load other portuguese custom BERT from local path, and this model isn't in huggingface, but in local with structure:

![image](https://user-images.githubusercontent.com/46280942/143885292-04c8d782-b9af-46a2-bb79-2ff0e6166005.png)

`pytorch_model, vocab.txt, config.json`

How could I load this model from local path in config settings?

**Comment by adrianeboyd** [comment, marked as answer]
Hi, the `name` can be anything that can be loaded by `AutoModel.from_pretrained`, including a path to a local directory rather than a model from the huggingface hub.

**Reply by leoitcode** [reply]
Thanks @adrianeboyd !!!

**Reply by leoitcode** [reply]
I'm not able to find this information in documentation, but thank you.

**Reply by adrianeboyd** [reply]
https://spacy.io/api/architectures#TransformerModel

If you have an architecture name like `spacy-transformers.TransformerModel.v1` you can usually find the right section in the docs through the search at the top right, where all the config parameters are documented.
